llama-index-integrations/tools/llama-index-tools-azure-code-interpreter/llama_index/tools/azure_code_interpreter/base.py

```diff
@@ -17,6 +18,8 @@
 
 from llama_index.core.tools.tool_spec.base import BaseToolSpec
 
+dotenv.load_dotenv()
```

Comment:
I don't think this should be in the tools class? It should be up to the user to use dotenv (or however they want to load env vars)

Reply:
@manekinekko yes, create-llama code will already use `dotenv`

Reply:
should I remove this line?

Reply:
yes this one and it's import, then this PR is good to go

Reply:
done.